pkgs/by-name/wa/wasabiwallet/package.nix

```diff
@@ -58,12 +58,18 @@ stdenv.mkDerivation rec {
 
   installPhase = ''
     mkdir -p $out/opt/${pname} $out/bin $out/share/applications
-    cp -Rv . $out/opt/${pname}
+    cp -Rv ./runner/work/WalletWasabi/WalletWasabi/build/linux-x64/* $out/opt/${pname}
 
-    makeWrapper "$out/opt/${pname}/wassabee" "$out/bin/${pname}" \
+    makeWrapper "$out/opt/${pname}/wassabee" "$out/bin/${pname}-desktop" \
       --suffix "LD_LIBRARY_PATH" : "${lib.makeLibraryPath runtimeLibs}"
 
-    makeWrapper "$out/opt/${pname}/wassabeed" "$out/bin/${pname}d" \
+    makeWrapper "$out/opt/${pname}/wassabeed" "$out/bin/${pname}-daemon" \
+      --suffix "LD_LIBRARY_PATH" : "${lib.makeLibraryPath runtimeLibs}"
+
+    makeWrapper "$out/opt/${pname}/wcoordinator" "$out/bin/${pname}-coordinator" \
+      --suffix "LD_LIBRARY_PATH" : "${lib.makeLibraryPath runtimeLibs}"
+
+    makeWrapper "$out/opt/${pname}/wbackend" "$out/bin/${pname}-backend" \
       --suffix "LD_LIBRARY_PATH" : "${lib.makeLibraryPath runtimeLibs}"
```

Comment:
Try to use a loop to avoid repeating many times.

Reply:
Done